tools/api-markdown-documenter/src/api-item-transforms/TransformApiPackage.ts

```diff
@@ -1,46 +0,0 @@
-/*!
- * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
- * Licensed under the MIT License.
- */
-import { ApiPackage } from "@microsoft/api-extractor-model";
-
-import { DocumentNode, SectionNode } from "../documentation-domain";
-import { apiItemToSections } from "./TransformApiItem";
-import { createDocument } from "./Utilities";
-import { ApiItemTransformationConfiguration } from "./configuration";
-import { createBreadcrumbParagraph, wrapInSection } from "./helpers";
-
-/**
- * Creates a {@link DocumentNode} for the specified `apiPackage`.
- *
- * @param apiPackage - The package content to be rendered.
- * @param config - See {@link MarkdownDocumenterConfiguration}.
- *
- * @returns The rendered Markdown document.
- */
-export function apiPackageToDocument(
```

Comment:
This logic moved into `api-item-transforms/TransformApiModel.ts`, and was made a private implementation detail.